airbyte-integrations/connectors/destination-elasticsearch/src/main/java/io/airbyte/integrations/destination/elasticsearch/ElasticsearchDestination.java

```diff
@@ -26,16 +25,12 @@ public class ElasticsearchDestination extends BaseConnector implements Destinati
   private final ObjectMapper mapper = new ObjectMapper();
 
   public static void main(String[] args) throws Exception {
-    final var destination = sshWrappedDestination();
+    final var destination = new ElasticsearchDestination();
```

Comment:
You likely don't want to edit here?

Reply:
oops i don't know what happened here. reverted all changes to elasticsearch